### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by naxoc**
Also clean up references to a file that was moved a while ago (the JsonIterator).

The mustache stuff was never used and the PHP.php file had no usages either.

### CODE REVIEW

File: publisher-specific-archive/LaSillaVaciaMigrator.php [hunk 1]
```diff
@@ -119,6 +119,7 @@ class LaSillaVaciaMigrator implements InterfaceCommand {
 					'children' => array(),
 				),
 				array(
+					'name'     => 'Red Paisa',
```
Comment (jorostoyanov) [inline publisher-specific-archive/LaSillaVaciaMigrator.php:122]: There is a duplicate line here :)

File: src/Utils/PHP.php
Comment (jorostoyanov) [inline src/Utils/PHP.php:1]: I see this being used in `src/Command/General/S3UploadsMigrator.php`.

**Review by jorostoyanov — CHANGES_REQUESTED** [review]
@naxoc Found one duplicate line and the PHP util class is still being used.

The rest looks good :)

File: src/Utils/PHP.php
Comment (naxoc) [reply src/Utils/PHP.php:1]: Great catch. I removed the usage

**Pull request merged**